### ISSUE TITLE
InaSAFE User Map - AIFDR projects

### ISSUE THREAD

**Issue by Charlotte-Morgan** [inasafe/django-user-map#3]
# Problem

There is an increasing desire for important people to know where Australian Govt funded projects are occurring.
# Proposal

That we consider a Project map that allows users to visually assess the alignment between different activites: ie the map might include information such as AIFDR priority provinces, locations where hazard data exist (by hazard), locations where contingency plans have been written (by hazard) +/- where trainers are available. I guess this list will get much longer.
# Purpose

Map users would be able to visually see alignment between different AIFDR activities & use it to explain programs to others, justify decisions & priorities
# Proposed solution

This additional functionality should be considered as a lower priority than the core user and event functions, but should be taken into consideration in the overall project design. That if this idea is supported, the initial map should address AIFDR needs not whole of DFAT in the first instance.
# cc

@akbargumbira @iyan31 @wonderchook @katechapman @davidr007 @griffij

**Comment by akbargumbira** [comment]
Hi, I don't think it's good to implement this on the same apps as this. By the way, for achieving this, my suggestion is just to make a 1-day-click-and-click-developed-apps using ArcGIS Builder apps and configure that apps to consume WFS/WMS for those maps (funded project, locations where hazard data exist, location where contingency plans have been writtern, etc). By doing that, it will not take much effort on development. 

Oh, @timlinux is gonna scream at me..

**Comment by Charlotte-Morgan** [comment]
@akbargumbira - when i worked in fire management I used to make maps in ArcGIS and publish as arc reader for all my 'customers'. So that is the way i would still do it if i did not have an @akbar :)

**Comment by akbargumbira** [comment]
No, it's different with ArcReader @Charlotte-Morgan. It's still a web application (The JS version will be coming soon http://www.esri.com/software/web-appbuilder). For the Flex/Silverlight it's available to use http://resources.arcgis.com/en/communities/flex-viewer/

**Comment by iyan31** [comment]
But....but....the subtitle of this project is **"A django application......."** :). On the other hand, I think it would be good if we can start from building the list using gdocs spreadsheet, then we can decide what platform will be used (an open source one I hope) :pray:

**Comment by akbargumbira** [comment]
But...but..the title of this project is InaSAFE User Map :). Sorry, I meant, the point is if we want to make this apps not specific to the requirements for user and event only (it can consume other maps using other service formats), we need to make a more general and neat architecture. Making it from scratch using Django and Leaflet, hmm, I am not sure it can be done within 2 months.

**Comment by timlinux** [comment]
> Hi, I don't think it's good to implement this on the same apps as this. By the way, for achieving this, my suggestion is just to make a 1-day-click-and-click-developed-apps using ArcGIS Builder apps and configure that apps to consume WFS/WMS for those maps (funded project, locations where hazard data exist, location where contingency plans have been writtern, etc). By doing that, it will not take much effort on development.

WAAAAAT! 

:-P But really @akbargumbira have you not see my awesome easy to publish QGIS Server mapping app. See http://demo.qgis.org and https://github.com/kartoza/docker-qgis-orchestration. The architecture should be very familiar to you and you can publish maps just by saving a .qgs project into a .btsync folder.

But yes if we are doing this in django it should be a separate app (under a mothership 'InaSAFE Django' project).
